Upstream: No
Reason: Allow extra leeway in rate limiting tests to prevent CI errors
--- a/pkg/middlewares/ratelimiter/rate_limiter_test.go
+++ b/pkg/middlewares/ratelimiter/rate_limiter_test.go
@@ -304,7 +304,7 @@ func TestRateLimit(t *testing.T) {
 			// we can expect some extra delay in addition to the rate limiting we already do,
 			// so we allow for some extra leeway there.
 			// Feel free to adjust wrt to the load on e.g. the CI.
-			minCount := wantCount * 95 / 100
+			minCount := wantCount * 85 / 100
 			if reqCount < minCount {
 				t.Fatalf("rate was slower than expected: %d requests (wanted > %d) in %v", reqCount, minCount, elapsed)
 			}
